apps/condo/domains/banking/utils/testSchema/index.js

```diff
@@ -239,7 +239,7 @@ async function updateTestBankIntegrationAccountContext (client, id, extraAttrs =
 async function createTestBankTransaction (client, account, contractorAccount, integrationContext, organization, extraAttrs = {}) {
     if (!client) throw new Error('no client')
     if (!account || !account.id) throw new Error('no account.id')
-    if (!contractorAccount || !contractorAccount.id) throw new Error('no contractorAccount.id')
+    if (contractorAccount && !contractorAccount.id) throw new Error('no contractorAccount.id')
```

Comment:
what if `contractorAccount` is undefined?

Reply:
why do we need to create transactions without a contractor?

Reply:
`BankTransaction` scheme allows you to leave this `contractorAccount` field empty.

And I managed to get such data from `sbbol`